### PR TITLE
Restyling - don't forget RTL languages..

### PR DESCRIPTION
**Pull request by matrix-amiel**
Hi @zbig01 .

Your work on restyling of the main screens is amazing.
I have some questions and points about it.
 
1. RTL languages -   
Basically not supposed to be problems with the rtl style as long as we styling with bootstrap. we load a bootsrap-rtl library and it makes a work, but still need to take a look on rtl before commit because always there are a little problems.
For example in the restyling of create encounter form (#1701) you use with 'text-right' class of bootstrap that makes text-align:right and it not match to RTL (I fixed it in this PR)

2.  Bootstrap standard -
I think we need  keep on the [grid principles of bootstrap ](https://getbootstrap.com/docs/3.3/css/#grid-intro) that requires that every class of 'col-\*-\*' be tested under div with class 'row', from my experience it's can makes trouble specially if we dream about responsive in the feature..   I would be happy to help with this refactor..

3. style_light and style_manila
I don't understand why only for those themes have scss files and why only both import a oe-bootstrap.scss file. I would be happy to explain. 

Thank you!
Amiel

### CODE REVIEW

**Comment by bradymiller** [comment]
hi @d3sandoval ,
Quick question; are the changes in interface/package-lock.json something that should be happening?

**Comment by d3sandoval** [comment]
@bradymiller yup! https://stackoverflow.com/questions/45022048/why-does-npm-install-rewrite-package-lock-json

**Comment by d3sandoval** [comment]
Relevant to this conversation, I've created a gist that explains a few approaches to modular scss: https://gist.github.com/d3sandoval/3b4437434cc89b261307461b46aa7a2b

More to come as I see more bootstrap changes rolling in and uncommitting vendor dependencies in #1600

File: interface/forms/newpatient/common.php [hunk 1]
```diff
@@ -227,7 +227,7 @@ function cancelClickedOld() {
                         <legend><?php echo xlt('Visit Details')?></legend>
                         <div id = "visit-details">
                             <div class="form-group ">
-                                    <label for="pc_catid" class="control-label col-sm-2 text-right"><?php echo xlt('Visit Category:'); ?></label>
+                                    <label for="pc_catid" class="control-label col-sm-2 text-to-side"><?php echo xlt('Visit Category:'); ?></label>
```
Comment (bradymiller) [inline interface/forms/newpatient/common.php]: hi @matrix-amiel ,
Wouldn't it make more sense to call this text-to-right?
(of which, it is assumed that rtl will do the opposite)
(then allowing another class, text-to-left that does the opposite)

Comment (bradymiller) [reply interface/forms/newpatient/common.php]: note i changed it to `oe-text-to-right`

Comment (matrix-amiel) [reply interface/forms/newpatient/common.php]: Agree.. done.

**Comment by bradymiller** [comment]
oh, I just realized my above post was in regards to another PR. Sorry for the mistaken post. -brady

**Comment by bradymiller** [comment]
Looking good. I'm going to bring this in (note it needs one more `npm run build`, but I'll do that in a commit after :) ). Thanks for fixing up RTL! -brady

**Pull request merged**